datalad/support/gitrepo.py

```diff
@@ -2129,10 +2197,9 @@ def call_git(self, args, files=None,
         ------
         CommandError if the call exits with a non-zero status.
         """
-        out, _ = self._git_custom_command(files, ["git"] + args,
-                                          expect_stderr=expect_stderr,
-                                          expect_fail=expect_fail,
-                                          check_fake_dates=True)
+        out, _ = self._call_git(args, files,
+                                expect_stderr=expect_stderr,
+                                expect_fail=expect_fail)
```

Comment:
Not clear to me, why `check_fake_dates` is gone here. True for other spots below, too.

Reply:
That is, because `_call_git()` always does it by default. And that, in turn, is because all usage of `_git_custom_command()`  by them had that setting. So this change is a no-change.

Reply:
Missed the default for `_call_git`, obv.